alerts.go

```diff
@@ -174,11 +186,19 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
 			statusMsg = color.RedString("CRITICAL")
 		case "WARNING":
 			statusMsg = color.YellowString("WARNING ")
+		case "UNKNOWN":
+			statusMsg = "UNKNOWN "
 		}
 	}
 	return fmt.Sprintf("%s %s %s %s%s", alert.ID, time.Unix(alert.OpenedAt, 0).Format(layout), statusMsg, monitorMsg, hostMsg)
 }
 
+var expressionTrimmer = regexp.MustCompile(`[\r\n]+\s*`)
+
+func trimExpression(expr string) string {
```

Comment:
To be precise, this func does not trim expression but make expression one line.
Perhaps, inlineExpression or something might be better as this function name...?